yaml/boundaries.yaml

```diff
@@ -264,6 +267,21 @@ filters:
     extra_columns: [scalerank]
     output:
       <<: *output_properties
+      <<: *ne_localized_kind_properties
       kind: region
       kind_detail: '4'
     table: ne
+  - filter:
+      featurecla:
+        - Unrecognized Admin-1 region boundary
```

Comment:
`Unrecognized Admin-1 region boundary` should output `unrecognized_macroregion` instead?

(Not that the data exists for that right now, but for correctness's sake.)

Reply:
Good point. Fixed in 84c00c04. (Although we don't seem to have support for `macroregion` from OSM boundaries either, so there might be problems at the transition.)